languages/go/config.json

```diff
@@ -51,7 +50,7 @@
         "slug": "comments",
         "uuid": "60186dfa-6a63-4859-a4c4-1e1ed1acf384",
         "concepts": ["comments"],
-        "prerequisites": [""]
+        "prerequisites": ["basics"]
```

Comment:
Alternative would be to let `comments` be without `prerequisite`, like `basics`. What do you think, @oanaOM?